metadata-ingestion/src/datahub/sdk/datajob.py

```diff
@@ -0,0 +1,257 @@
+        if is_ingestion_attribution():
+            editable_props = self._get_editable_props()
+            if editable_props is not None and editable_props.description is not None:
+                warnings.warn(
+                    "Overwriting non-ingestion description from ingestion is an anti-pattern.",
+                    category=IngestionAttributionWarning,
+                    stacklevel=2,
+                )
+                # Force the ingestion description to show up.
+                editable_props.description = None
+
+            self._ensure_datajob_props().description = description
+        else:
+            self._ensure_editable_props().description = description
+
+    @property
+    def name(self) -> str:
+        """Get the name of the data job."""
+        return self.urn.job_id
+
+    @property
+    def display_name(self) -> Optional[str]:
+        """Get the display name of the data job."""
+        return self._ensure_datajob_props().name
+
+    def set_display_name(self, display_name: str) -> None:
+        """Set the display name of the data job."""
+        self._ensure_datajob_props().name = display_name
+
+    @property
+    def external_url(self) -> Optional[str]:
+        """Get the external URL of the data job."""
+        return self._ensure_datajob_props().externalUrl
+
+    def set_external_url(self, external_url: str) -> None:
+        """Set the external URL of the data job."""
+        self._ensure_datajob_props().externalUrl = external_url
+
+    @property
+    def custom_properties(self) -> Dict[str, str]:
+        """Get the custom properties of the data job."""
+        return self._ensure_datajob_props().customProperties
+
+    def set_custom_properties(self, custom_properties: Dict[str, str]) -> None:
+        """Set the custom properties of the data job."""
+        self._ensure_datajob_props().customProperties = custom_properties
+
+    @property
+    def created(self) -> Optional[datetime]:
+        """Get the creation timestamp of the data job."""
+        return parse_time_stamp(self._ensure_datajob_props().created)
+
+    def set_created(self, created: datetime) -> None:
+        """Set the creation timestamp of the data job."""
+        self._ensure_datajob_props().created = make_time_stamp(created)
+
+    @property
+    def last_modified(self) -> Optional[datetime]:
+        """Get the last modification timestamp of the data job."""
+        return parse_time_stamp(self._ensure_datajob_props().lastModified)
+
+    def set_last_modified(self, last_modified: datetime) -> None:
+        """Set the last modification timestamp of the data job."""
+        self._ensure_datajob_props().lastModified = make_time_stamp(last_modified)
+
+    @property
+    def flow_urn(self) -> DataFlowUrn:
+        """Get the data flow associated with the data job."""
+        return self.urn.get_data_flow_urn()
+
+    def _set_browse_path_from_flow(self, flow: DataFlow) -> None:
```

Comment:
This creates browsepaths like this when there's a container for dataflow 
<img width="1279" alt="Screenshot 2025-05-22 at 5 25 28 PM" src="https://github.com/user-attachments/assets/2d85d26d-8986-40e9-8a1c-c83c42950390" />
<img width="1275" alt="Screenshot 2025-05-22 at 5 25 35 PM" src="https://github.com/user-attachments/assets/88815333-7c0e-494f-bdf4-eedfc162a3d6" />

Reply:
when there's no container, it just creates this 
![Uploading Screenshot 2025-05-22 at 5.29.08 PM.png…]()